Assets/Mirror/Editor/Weaver/Mirror.Weaver.asmdef

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Mirror.Weaver",
+    "references": [],
+    "optionalUnityReferences": [],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": true,
+    "overrideReferences": false,
+    "precompiledReferences": [],
```

Comment:
can you use the precompiledReferences like they do?  then we don't have to include Cecil anymore

Reply:
mono.cecil dlls aren't in unity 2018. only in 2019 as it seems.

Reply:
This is in 2018.3.8:
```
./Unity.app/Contents/il2cpp/build/Mono.Cecil.Rocks.dll
./Unity.app/Contents/il2cpp/build/il2cppcore/Mono.Cecil.Rocks.dll
./Unity.app/Contents/il2cpp/build/il2cppcore/Mono.Cecil.dll
./Unity.app/Contents/il2cpp/build/il2cppcore/Mono.Cecil.Pdb.dll
./Unity.app/Contents/il2cpp/build/il2cppcore/Mono.Cecil.Mdb.dll
./Unity.app/Contents/il2cpp/build/Mono.Cecil.dll
./Unity.app/Contents/il2cpp/build/Mono.Cecil.Pdb.dll
./Unity.app/Contents/il2cpp/build/Mono.Cecil.Mdb.dll
./Unity.app/Contents/Mono/lib/mono/unity/Mono.Cecil.dll
./Unity.app/Contents/Mono/lib/mono/2.0/Mono.Cecil.dll
./Unity.app/Contents/Mono/lib/mono/unity_web/Mono.Cecil.dll
```

Of course they include cecil.  They shipped with the deprecated HLAPI which does include cecil.

Reply:
adding this to asmdef:
`
    "precompiledReferences": [
        "Mono.Cecil.dll",
        "Mono.Cecil.Mdb.dll",
        "Mono.Cecil.Pdb.dll"
    ],
`
causes an error when selecting the asmdef file in Unity:
AssemblyDefinitionException: Referenced precompiled assembly 'Mono.Cecil.dll' not found
UnityEditor.AssemblyDefinitionImporterInspector.LoadAssemblyDefintionState (System.String path) (at /Users/builduser/buildslave/unity/build/Editor/Mono/Inspector/AssemblyDefinitionImporterInspector.cs:740)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)